cli/tests/pcluster/config/defaults.py

```diff
@@ -136,6 +135,48 @@
     "fsx_settings": None,
     "dcv_settings": None,
     "cw_log_settings": None,
+    "cluster_config_metadata": {"sections": {}},
+    "architecture": "x86_64",
+}
+
+DEFAULT_CLUSTER_HIT_DICT = {
```

Comment:
We can define a "COMMON_CLUSTER_DICT" and create the other ones by concatenating the common and the custom stuff.

Reply:
Replied below